stable/oauth2-proxy/README.md

```diff
@@ -104,6 +104,9 @@ Parameter | Description | Default
 `service.clusterIP` | cluster ip address | `nil`
 `service.loadBalancerIP` | ip of load balancer | `nil`
 `service.loadBalancerSourceRanges` | allowed source ranges in load balancer | `nil`
+`serviceAccount.create` | create a service account | `true`
```

Comment:
You state the opposite in values.yaml. ;)
actually, it can be enabled by default, but this is a breaking change and should be advertised as such (with major version bump and paragraph about how to upgrade)

Reply:
Done.